### PR TITLE
search notes of MR up to 300

### PR DESCRIPTION
**Pull request by kitagry**
search notes up to 300 to update note.

### CODE REVIEW

File: src/notifier/gitlab.rs [hunk 1]
```diff
@@ -4,14 +4,15 @@ use crate::template::Template;
 use anyhow::{Context, Result};
 use gitlab::api::projects::merge_requests::notes::{EditMergeRequestNote, MergeRequestNotes};
 use gitlab::api::{self, projects::merge_requests::notes::CreateMergeRequestNote, Query};
-use gitlab::types::Note;
```
Comment (kitagry) [inline src/notifier/gitlab.rs:7]: types module was deleted in https://gitlab.kitware.com/utils/rust-gitlab/-/merge_requests/453

And, in docs, we should declare own struct in order to fetch what we need. https://docs.rs/gitlab/0.1700.1/gitlab/api/index.html

**Review by hirosassa — APPROVED** [review]
Thank you for following latest dependencies!

**Comment by hirosassa** [comment]
@kitagry Released v1.3.0
https://github.com/hirosassa/ksnotify/releases/tag/v1.3.0

**Comment by kitagry** [comment]
Thank you!

**Pull request merged**